### ISSUE TITLE
G  4.0 lineDash 设置了后，无法取消

### ISSUE THREAD

**Issue by dxq613** [antvis/g#206]
- [x] I have searched the [issues](https://github.com/antvis/g/issues) of this repository and believe that this is not a duplicate.

### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/g-canvas-linedash-1zuzx)

[https://github.com/antvis/g/blob/4.x/packages/g-canvas/src/util/draw.ts#L22](https://github.com/antvis/g/blob/4.x/packages/g-canvas/src/util/draw.ts#L22)

### Steps to reproduce
shape.attr('lineDash', [2, 2]); 后 调用 shape.attr('lineDash', null);  报错

### What is expected?
调用 shape.attr('lineDash', null);  不再出现 lineDash 效果

### What is actually happening?
报错

| Environment | Info |
|---|---|
| g | 4.0 |
| System | 任意 |
| Browser | chrome |

**Comment by dengfuping** [comment]
- 重置 `lineDash` 的推荐做法是置为空数组:
```js
shape.attr('lineDash', []);
```
- 不过应该也要兼容以下的写法:
```js
shape.attr('lineDash', null);
shape.attr('lineDash', undefined);
shape.attr('lineDash', false);
shape.attr('lineDash', 0);
shape.attr('lineDash', {});
shape.attr('lineDash', 'test');
```

**Comment by dengfuping** [comment]
- Closed by #208.
